### PR TITLE
[FLOSS H2] doc: make man page key names version number consistent

### PR DESCRIPTION
**Pull request by JakobWonisch**
## Basics

These points need to be fulfilled for every PR:

- [x] Short descriptions of your changes are in the release notes
      (added as entry in `doc/news/_preparation_next_release.md` which
      contains `_(my name)_`)
      **Please always add something to the release notes.**
- [x] Details of what you changed are in commit messages
      (first line should have `module: short statement` syntax)
- [x] References to issues, e.g. `close #X`, are in the commit messages.
- [ ] The buildservers are happy. If not, fix **in this order**:
  - [ ] add a line in `doc/news/_preparation_next_release.md`
  - [ ] reformat the code with `scripts/dev/reformat-all`
  - [ ] make all unit tests pass
  - [ ] fix all memleaks
- [x] The PR is rebased with current master.

If you have any troubles fulfilling these criteria, please write
about the trouble as comment in the PR. We will help you.
But we cannot accept PRs that do not fulfill the basics.

## Checklist

Check relevant points but **please do not remove entries**.
For docu fixes, spell checking, and similar none of these points below
need to be checked.

- [ ] I added unit tests for my code
- [ ] I fully described what my PR does in the documentation
      (not in the PR description)
- [ ] I fixed all affected documentation
- [ ] I added code comments, logging, and assertions as appropriate (see [Coding Guidelines](https://master.libelektra.org/doc/CODING.md))
- [ ] I updated all meta data (e.g. README.md of plugins and [METADATA.ini](https://master.libelektra.org/doc/METADATA.ini))
- [ ] I mentioned every code not directly written by me in [THIRD-PARTY-LICENSES](doc/THIRD-PARTY-LICENSES)

## Review

Reviewers will usually check the following:

- [ ] Documentation is introductory, concise, good to read and describes everything what the PR does
- [ ] Examples are well chosen and understandable
- [ ] Code is conforming to [our Coding Guidelines](https://master.libelektra.org/doc/CODING.md)
- [ ] APIs are conforming to [our Design Guidelines](https://master.libelektra.org/doc/DESIGN.md)
- [ ] Code is consistent to [our Design Decisions](https://master.libelektra.org/doc/decisions)

## Labels

If you are already Elektra developer:

- Add the "work in progress" label if you do not want the PR to be reviewed yet.
- Add the "ready to merge" label **if the basics are fulfilled** and you also
  say that everything is ready to be merged.

### CODE REVIEW

**Comment by markus2330** [comment]
Please only set "ready to merge" if the build server is already happy.

**Review by markus2330 — CHANGES_REQUESTED** [review]
This PR modifies a generated file. It was generated from doc/help/elektra-key-names.md, please fix also this file. (doc/man/man7/elektra-key-names.7 should be updated automatically after recompiliation if ronn is installed).

**Review by flo91 — CHANGES_REQUESTED** [review]
**[FLOSS H2]**
Thank you for the PR, great finding! :+1:
Currently two buildserver tests are still failing, but it looks like that's a general problem of the build system, because the same two checks also recently failed on many other PRs.

I'm not really sure if the commit 8bd1ca4e6df60b55a8f69f710ca40620a64ffff4 with the `DartConfiguraton.tcl` file is useful.
That file contains private paths like `/media/storage/Uni/FLOSS/libelektra-myfork` which don't look like something that should be in a public repository.


I also noticed some small points for improvement:

- Please fill the checkboxes for the fulfilled points in the categorie _Basics_


If the checklist for PRs is followed, two points of the category _Basics_ need attention:
- **Reference to issue**
	Most of the PRs for the FLOSS H2 task don't have references to issues.
	I'm not really sure if this point applies to our PRs.
	Shall we create new issues for the changes we made?

- **The PR is rebased with current master.**
In https://www.libelektra.org/devgettingstarted/contributing#creating-a-pull-request is stated in point 6: _keep in mind to sync your fork - fetch, rebase & push_
When I open the repository for the PR (https://github.com/JakobWonisch/libelektra/tree/man-key-names-consistency), there is the message _This branch is 4 commits ahead, 11 commits behind ElektraInitiative:master._ So it looks like this has still to be done.

File: DartConfiguration.tcl [hunk 1]
```diff
@@ -0,0 +1,105 @@
+# This file is configured by CMake automatically as DartConfiguration.tcl
```
Comment (markus2330) [inline DartConfiguration.tcl:1]: Is the inclusion of this file on purpose?

**Pull request merged**